manifests/b/Badlion/BadlionClient/4.3.0/Badlion.BadlionClient.locale.en-US.yaml

```diff
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Badlion.BadlionClient
+PackageVersion: 4.3.0
+PackageLocale: en-US
+Publisher: Badlion
+PublisherSupportUrl: https://support.badlion.net
+PackageName: Badlion Client
+License: Proprietary
+Copyright: © 2013-2023 ESL Gaming Online, Inc.
+ShortDescription: Badlion Client Minecraft Launcher
+Description: |-
+  Badlion Client is a Minecraft launcher with many added features.
+  It includes anticheat, built-in FPS boost and other mods, and is compatible with most versions of minecraft 1.7 or newer.
+Moniker: blc
+Tags:
+- minecraft
+- optifine
+Documentations:
+- DocumentLabel: FAQ
+  DocumentUrl: https://client.badlion.net/faq

```

Comment:
```suggestion
#   DocumentUrl: 
```

This might be due to a network block of data centers, as it loads normally for me.